src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -106,8 +106,13 @@ export default function connectRatingMenu(renderFn, unmountFn = noop) {
     return {
       $$type: 'ais.ratingMenu',
 
-      getConfiguration() {
-        return { disjunctiveFacets: [attribute] };
+      getConfiguration(state) {
+        return state.setQueryParameters({
+          disjunctiveFacets: [attribute],
```

Comment:
See if we want to support previous facets at this level instead of in `enhanceConfiguration`.

Reply:
https://github.com/algolia/instantsearch.js/pull/3976#discussion_r308155858